lib/api/utilities/functions.php

```diff
@@ -6,22 +6,26 @@
  * loaded automatically when the Beans framework is included.
  *
  * @package API\Utilities
+ *
+ * @since   1.5.0
  */
 
+// Load the dependencies.
+require_once dirname( __FILE__ ) . '/polyfills.php';
```

Comment:
Let's keept the loaders together by moving this one [over there](https://github.com/Getbeans/Beans/blob/development/lib/api/init.php#L36).